pkg/v1/sdk/capabilities/hack/generate-package-secret.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 VMware, Inc. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -euo pipefail
+
+MY_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+TF_ROOT="${MY_DIR}/../../../../../"
+
+# Always run from capabilities directory for reproducibility
+cd "${MY_DIR}/.."
+
+# Run YTT, passing the arguments to this script straight through to YTT to ease future extensions template
+ytt -f ./hack/ytt/schema.yaml -f ./hack/ytt/default-package-secret.yaml -f ./hack/ytt/values.yaml $@
```

Comment:
question: Once the `Secret` manifests get generated, I am curious on how it gets applied to a cluster (Because we want to make it available in a cluster under system namespace so there won't be `NotFound` error when `clusterbootstrap_controller` processes the `ClusterBootstrapTemplate`)